### ISSUE TITLE
Current default rendering of segments looks a bit "flat"

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#1196]
Would it be possible to have a bit more "texture/shadow" effects for the segments rendering?

<img width="866" alt="image" src="https://github.com/user-attachments/assets/5fd983b3-6265-4a23-93c3-3b4c07f9d9dc">

**Comment by ekatrukha** [comment]
I am exploring/playing a bit with some other volume rendering methods, like attenuated MIP, MINA, 
but this part is not ready for release yet.

We can also try to introduce lighting, so there is a shade.

**Comment by tischi** [comment]
Sounds great, "but" the rendering of one of the (I think binary) intensity images did actually look much more 3-D....do you know what the difference is?

![image](https://github.com/user-attachments/assets/eddc033a-c10b-4516-9aa3-606c768fa473)

**Comment by ekatrukha** [comment]
Interesting. 
Can you please tell me label number and dataset.
And possibly attach a screenshot of brightness and contrast with alpha values panel extended?

**Comment by tischi** [comment]
Open PlaytBrowser

<img width="727" alt="image" src="https://github.com/user-attachments/assets/fc6cb48e-d9d2-4df2-9ca6-9753495de353">

<img width="1095" alt="image" src="https://github.com/user-attachments/assets/3776ddd7-718f-4965-9556-f086100c1b6a">

**Comment by ekatrukha** [comment]
On the screenshot, you have extended the alpha slider paned for the groups,
you need to extend it above Groups label (three triangles), for the specific source.

**Comment by tischi** [comment]
Thanks:

<img width="1123" alt="image" src="https://github.com/user-attachments/assets/ede3c8af-1907-4578-91cb-d259d1bb0926">

**Comment by tischi** [comment]
<img width="1123" alt="image" src="https://github.com/user-attachments/assets/c9c2f2b7-c91e-4b7d-bd52-f09fae78005b">


Seems like putting the Y_alpha to low values creates a flat rendering.

**Comment by tischi** [comment]
However, I tried to play with those sliders when rendering some of the  tissue segmentation labels and I could not reproduce this nice appearance. But you may be more successful as you better understand what you are doing. I was just randomly moving the sliders. 

Is there somewhere a documentation/explanation what those sliders do?

**Comment by ekatrukha** [comment]
Ok, but this `prospr` value `glt1` from your example is not loaded to BVV as an annotation, rather like a volume.
Is it supposed to be this way?

Also, how can I show/select only two segmentations? What is the sequence of actions?
To get a picture like you showed in your first comment here.

For now, I tried to `Cntr` + `left click` on the table, Mobie selects one segmentation.
I see that for BVV LUT it assigns alpha value of 1 (or 255) for the selected segmentation, while for the rest it is 0.075 (19), 
so non-selected segmentations are still present in the BVV view. 
Is it ok? They would be visible, since alpha is non-zero.

As for the what sliders do, the top slider assigns colors of LUT to each voxel.
Depending on the intensity of the voxel, it will get a color from LUT.
The second changes the gamma for this color mapping.
The third slider assigns alpha value (opacity) to the voxel, again, depending on its intensity.
Opacity zero = voxel transparent. Opacity 1 = voxel fully opaque.
Imaging you have unsigned bit (256) image.
If this alpha range slider settings are from 100 to 200, then voxels with intensity 100 or less will be rendered fully transparent (opacity = 0). Voxels with intensity from 200 to 255 would be rendered fully opaque (opacity 1).
Voxels with intensity from 100 to 200 -> opacity will linearly increase from 0 to 1 in this range.

If the fourth slider value is not 1, instead of linear incread iw would be a power function with its value (like a standard gamma correction).
Hope it helps.

**Comment by ekatrukha** [comment]
Ok, I got it.
When using `prospr` / `glt1` view, it loads it as a volume, 
so it does trilinear interpolation, rather than `floor`/nearest neighbor.
Then the values at the surface of the blob will have intermediate value 
between mask 255 and 0 background. 
This is what creates such a nice "shady" look :)
Basically, it does what a "surface render" will do, kind of.

**Comment by tischi** [comment]
OK, interesting! Thanks for looking in to this!

> For now, I tried to Cntr + left click on the table, Mobie selects one segmentation.
I see that for BVV LUT it assigns alpha value of 1 (or 255) for the selected segmentation, while for the rest it is 0.075 (19),
so non-selected segmentations are still present in the BVV view.
Is it ok? They would be visible, since alpha is non-zero.

You can Ctrl+Left-Click on segments the BDV window to select multiple labels.
You can change the opacity of non-selected labels in the Settings wheel of the segmentation image.

Shall we close the issue? Or shall we leave it open until, maybe, in some future, some more fancy rendering modes might become available. I would tend to leave it open, because users may have a similar experience and then we can point them here.

**Comment by ekatrukha** [comment]
Yes, let's keep it open. It will also motivate me to work on adding new rendering methods.
